<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="Form">
      <item row="0" column="0">
       <widget class="QLabel" name="sum_label">
        <property name="toolTip">
         <string>Початкова сума</string>
        </property>
        <property name="text">
         <string>Сума</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="interest_rate_label">
        <property name="text">
         <string>% ставка</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="storage_period_label">
        <property name="text">
         <string>Терімн Зберігання</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="sum_edit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="interest_rate_edit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="storage_period_edit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Result">
      <item>
       <layout class="QHBoxLayout" name="Profit">
        <item>
         <widget class="QLabel" name="profit_label">
          <property name="text">
           <string>Прибуток</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="profit_value">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Total_sum">
        <item>
         <widget class="QLabel" name="total_sum_label">
          <property name="text">
           <string>Загальна сума</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="total_sum_value">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Buttons">
      <item>
       <widget class="QPushButton" name="Calculate">
        <property name="text">
         <string>Обчислити</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Clear">
        <property name="text">
         <string>Очистити поля</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Quit">
        <property name="text">
         <string>Закрити програму</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
